index.md

```diff
@@ -0,0 +1,222 @@
+
+```sh
+curl https://wasmtime.dev/install.sh -sSf | bash
+```
+
+You can also download binaries directly from the [GitHub
+Releases](https://github.com/bytecodealliance/wasmtime/releases) page.
+
+</div>
+
+<div id="platform-instructions-win" class="dn">
+
+It looks like you’re running Windows. To install Wasmtime, download and run the
+following, and then follow the onscreen instructions.
+
+[Wasmtime Installer](https://github.com/bytecodealliance/wasmtime/releases/download/dev/wasmtime-dev-x86_64-windows.msi)
+
+If you’re a Windows Subsystem for Linux user run the following in your terminal,
+then follow the on-screen instructions to install Wasmtime.
+
+```sh
+curl https://wasmtime.dev/install.sh -sSf | bash
+```
+
+</div>
+
+<div id="platform-instructions-default" class="db">
+
+If you’re running macOS or Linux, download and install Wasmtime by running the
+following in your terminal and following the on-screen instructions.
+
+```sh
+curl https://wasmtime.dev/install.sh -sSf | bash
+```
+
+--------------------------------------------------------------------------------
+
+If you are running Windows, download and run the [Wasmtime
+Installer](https://github.com/bytecodealliance/wasmtime/releases/download/dev/wasmtime-dev-x86_64-windows.msi)
+then follow the on-screen instructions.
+
+--------------------------------------------------------------------------------
+
+Alternatively, you can download releases for all supported platforms from the
+[Wasmtime GitHub page](https://github.com/bytecodealliance/wasmtime/releases).
+
+</div>
+
+</div>
+</section>
+
+<section class="section-tinted">
+<div class="container w-container">
+
+## Example
+
+If you've got the [Rust compiler
+installed](https://www.rust-lang.org/tools/install) then you can take some Rust
+source code:
+
+```rust
+fn main() {
+    println!("Hello, world!");
+}
+```
+
+and compile/run it with:
+
+```sh
+$ rustup target add wasm32-wasi
+$ rustc hello.rs --target wasm32-wasi
+$ wasmtime hello.wasm
+Hello, world!
+```
+
+</div>
+</section>
+
+<section>
+<div class="container w-container">
+
+## Features
+
+* **Fast**. Wasmtime is built on the optimizing [Cranelift] code generator to
+  quickly generate high-quality machine code either at runtime or
+  ahead-of-time. Wasmtime's runtime is also optimized for cases such as
+  efficient instantiation, low-overhead transitions between the embedder and
+  wasm, and scalability of concurrent instances.
+
+* **[Secure]**. Wasmtime's development is strongly focused on the correctness of
+  its implementation with 24/7 fuzzing donated by [Google's OSS Fuzz],
+  leveraging Rust's API and runtime safety guarantees, careful design of
+  features and APIs through an [RFC process], a [security policy] in place
+  for when things go wrong, and a [release policy] for patching older versions
+  as well. We follow best practices for defense-in-depth and known
+  protections and mitigations for issues like Spectre. Finally, we're working
```

Comment:
I'm finding this first sentence pretty tough to read. I would re-work it or break it up a bit. Maybe something like:

```suggestion
* **[Secure]**. Wasmtime's development is strongly focused on correctness and
  security. Building on top of Rust's runtime safety guarantees, each Wasmtime
  feature goes through careful review and consideration via an [RFC process]. Once
  features are designed and implemented, they undergo 24/7 fuzzing donated by
  [Google's OSS Fuzz]. As features stabilize they become part of a [release][release policy],
  and when things go wrong we have a well-defined [security policy] in place to quickly
  mitigate and patch any issues. We follow best practices for defense-in-depth and
  integrate protections and mitigations for issues like Spectre. Finally, we're working
  its implementation with 24/7 fuzzing donated by [Google's OSS Fuzz],
  leveraging Rust's API and runtime safety guarantees, careful design of
  features and APIs through an [RFC process], a [security policy] in place
  for when things go wrong, and a [release policy] for patching older versions
  as well. We follow best practices for defense-in-depth and known
  protections and mitigations for issues like Spectre. Finally, we're working
```